pkgs/development/python-modules/bottombar/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, flit-core
+}:
+
+buildPythonPackage rec {
+  pname = "bottombar";
+  version = "1.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "evalf";
+    repo = "bottombar";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-/3m34HcYmmEf92H3938dYV1Q6k44KaCb9TDx9nDNPnM=";
+  };
+
+  propagatedBuildInputs = [
+    flit-core
+  ];
```

```suggestion
  nativeBuildInputs = [
    flit-core
  ];
```

That is a build tool.